pl-pseudo-crepes.md

```diff
@@ -0,0 +1,57 @@
+# Pseudo crepes
+
+## Składniki
+
+### Ciasto
+
+-   3 średnie jaja,
+-   mąka pszenna typu 00/babuni 200g,
+-   chłodna woda z kranu 200ml,
+-   mleko 200ml,
+-   drożdże świeże lub suche 1gr,
+-   sól 3g,
+-   cukier biały 10g,
+-   masło 30g + tyle co do smarowania patelni,
+-   ekstrakt waniliowy łyżeczka do herbaty,
+
+### Utensylia
+-   coś do mieszania - najlepiej trzepaczka
+-   miska do przygotowania ciasta
+-   chochelka
+-   patelnia do smażenia - byle by nie najmniejsza
+
+### Dodatki
+
+* do smaku
+
+## Przygotowanie
+
+### Smażenie
```

Comment:
NIT: Tutaj można się zastanowić czy potrzebne to info

Reply:
co racja to racja